js/framework/AnalyticsConstants.js

```diff
@@ -819,21 +885,188 @@ if (!OO.Analytics.EVENT_DATA)
     var checkAdErrorData = OO._.bind(checkDataType, this, "AdErrorData");
     this.error = checkAdErrorData(error, "error", ["string", "object"]);
   };
+  
+  /**
+   * @public
+   * @class Analytics.EVENT_DATA#AdRequestData
+   * @classdesc Contains information about the ad request event. 
+   * @property {string} adPluginName The name of the ad plugin used
+   * @property {number} adPosition The position the ad is scheduled to play
```

Comment:
Change to:
The position the ad that is scheduled to play

Reply:
OK actually cancel that change. I think it should be something like 
The position, in seconds, the ad is scheduled to play

Trying to clarify that it isn't a position like "preroll" or "midroll"